Dockerfile

```diff
@@ -0,0 +1,24 @@
+# Basic docker based environment
+# Necessary to trick dokku into building the documentation
+# using dockerfile instead of herokuish
+FROM ubuntu:17.04
```

Comment:
Any particular reason for not using 18.04 LTS ?

Reply:
Because I'm lazy and was copy/pasting like a boss... :blush:

Reply:
The world thrive on copy/pasting :smile: